### ISSUE TITLE
Heroes 2, button positioning and button spacing in windows

### ISSUE THREAD

**Issue by LeHerosInconnu** [ihhub/fheroes2#1355]
In fheroes2, in the windows, the position of the buttons "YES" and "NO" is not the same as in the original game.

In the original game, the buttons are positioned more harmoniously in the window.
The horizontal space between the buttons has a value of three sevenths of the space to be distributed, and the horizontal space between the buttons and the edge of the window has a value of two sevenths of the space to be distributed.
In fheroes2, the buttons are horizontally too spaced between them and horizontally placed too close to the window edge, and the horizontal distance between the buttons and the window edges is not the same for each button.
In short, it is not very harmonious.
I suggest applying the rule from Heroes 2 for all similar windows in fheroes2.

In the original game:

![Graveyard search the graves 01](https://user-images.githubusercontent.com/43583994/91292170-6bec9980-e796-11ea-9e71-4a1f9d1fb48a.png)

In fheroes2:

![Graveyard search the graves 02](https://user-images.githubusercontent.com/43583994/91292181-71e27a80-e796-11ea-8ba1-1d1b0b20866c.png)
